etrago/tools/constraints.py

```diff
@@ -1231,33 +1244,18 @@ def snapshot_clustering_seasonal_storage_nmp(self, n, sns):
 
     dispatch =  get_var(n, c, 'p_dispatch').loc[last_hour].set_index(candidates)
     store = get_var(n, c, 'p_store').loc[last_hour].set_index(candidates)
-    last_soc_total = soc_total.loc[last_hour].set_index(candidates)
-    last_soc_inter = soc_inter.shift(1).fillna(soc_inter.loc[candidates[-1]])
+    last_soc_intra = soc_intra.loc[last_hour].set_index(candidates)
 
     coeff_var = [(-1, next_soc_inter),
                  (eff_stand.pow(24), soc_inter),
-                 (eff_stand, last_soc_total),
-                 (-eff_stand, last_soc_inter),
+                 (eff_stand, last_soc_intra),
                  (-1/eff_dispatch, dispatch),
```

Comment:
Do we need to change p_dispatch and p_store to the values of the next hour which is the first hour of the next representative day? The soc_intra is defined that way in this code. Is this just a definition question or do we have to align this constraint to avoid mixed definitions?